implementations/contracts/interfaces/IERC725Y.sol

```diff
@@ -14,8 +14,9 @@ interface IERC725Y is IERC165 {
     /**
      * @notice Emitted when data at a key is changed
      * @param dataKey The data key which value is set
+     * @param dataValue The data value which is set to the key
      */
-    event DataChanged(bytes32 indexed dataKey);
+    event DataChanged(bytes32 indexed dataKey, bytes indexed dataValue);
```

Comment:
We don't want to index the `dataValue`. Otherwise it will cost more gas: https://www.reddit.com/r/ethdev/comments/78ugki/do_indexed_parameters_in_events_cost_more_gas/

```suggestion
    event DataChanged(bytes32 indexed dataKey, bytes dataValue);
```